services/src/test/scala/cromwell/pipeline/service/AggregationServiceTestImpl.scala

```diff
@@ -0,0 +1,21 @@
+package cromwell.pipeline.service
+import cromwell.pipeline.datastorage.dto._
+
+import scala.concurrent.Future
+
+class AggregationServiceTestImpl(valToReturn: DummyToReturn) extends AggregationService {
+
+  override def aggregate(run: Run): Future[CromwellInput] =
+    valToReturn match {
+      case WithException(exc)                   => Future.failed(exc)
```

Comment:
Why are you writing 'exc' instead for 'exception' or 'error' ?

Reply:
Takes less space ) I thought that "exc" is a well known and quite widespread abbreviation.

Reply:
you can write just 'e' but however.